libs/feature/catalog/src/lib/organisations/service/organisations-from-metadata.service.ts

```diff
@@ -163,6 +164,29 @@ export class OrganisationsFromMetadataService
     })
   }
 
+  private hydrateResourceContactWithOrg(
+    metadataRecord: MetadataRecord,
+    organisation: Organisation
+  ): MetadataRecord {
+    const firstResourceContact = metadataRecord.resourceContacts[0]
+    const logoUrl =
+      firstResourceContact.logoUrl ||
+      (organisation.logoUrl ? getAsUrl(`${organisation.logoUrl}`) : null)
+    const mappedOrg = {
+      name: organisation.name,
+      organisation: organisation.name,
+      email: organisation.email || firstResourceContact.email,
+      logoUrl: logoUrl,
+      website: metadataRecord.resourceContacts[0].website,
+    } as MetadataContact
+
+    metadataRecord.resourceContacts = [
```

Comment:
I wouldn't add a new contact like for the `group` strategy, that was my point in the previous message.
BTW if you do that the name of the function is not meaningful.

The `group` strategy add a new contact from the group of the record (which could be different from the recource contact), while the `metadata` strategy just hydrates the first existing resource contact.